app/scripts/migrations/105.ts

```diff
@@ -103,6 +103,7 @@ function createInternalAccountsForAccountsController(
           // initial updateAccounts call.
           type: 'HD Key Tree',
         },
+        importTime: Date.now(),
```

Comment:
Looks like this migration change, and the 119 migration change, were caused by a dependency bump that we can remove from the lock file